src/tests/ngraph_helpers/ngraph_functions/src/scatter_elements_update.cpp

```diff
@@ -20,5 +20,21 @@ std::shared_ptr<ngraph::Node> makeScatterElementsUpdate(const ngraph::Output<Nod
     return dtsNode;
 }
 
+std::shared_ptr<ngraph::Node> makeScatterElementsUpdate(const ngraph::Output<Node> &in,
+                                                        const element::Type& indicesType,
+                                                        const std::vector<size_t>& indicesShape,
+                                                        const std::vector<int64_t>& indices,
+                                                        const ngraph::Output<Node> &update,
+                                                        int axis,
+                                                        ov::op::v12::ScatterElementsUpdate::Reduction reduceMode,
+                                                        bool useInitVal
+                                                        ) {
+    auto indicesNode = std::make_shared<ngraph::opset1::Constant>(indicesType, indicesShape, indices);
```

Comment:
I think this operation can be created in-line in the code.

Reply:
Done. 
Operation creation moved to ScatterElementsUpdate12LayerTest::SetUp(), ngraph interface is not in use anymore see - [SetUp() method](https://github.com/openvinotoolkit/openvino/pull/19254/files#diff-5002e9f2ae0cc251307cbb256cec4445d3211509f5e2c1184d8136d58330afb7R111) .